addons/OPT/SHOP/fn_order.sqf

```diff
@@ -0,0 +1,110 @@
+/**
+* Description:Kauf ausführen und Fahrzeug erstellen.
+* 
+* 
+* Author:
+* [GNC]Lord-MDB
+*
+* Arguments:
+* 
+*
+* Return value:
+* 
+*
+* Server Only:
+* No
+* 
+* Global:
+* No
+* 
+* API:
+* No
+* 
+* Example:
+* 
+*/
+
+#include "macros.hpp"
+
+params 
+[
+    ["_Datensatz", []], 
+    ["_box", objNull],
+    ["_moveInVeh", nil],
+	["_unitCost", 0]
+];
+
+private _airRaktenmagazin = [];
+private _airRaktenweapon = [];
+private _airGunmagazin = [];
+private _airGunweapon = [];
+private _vehMagazin = [];
+private _vehWeapon = [];
+private _pylon = [];
+private _raketencontrol = [];
+private _zusatz = [];
+private _waffenkosten = 0;
+private _weaponold = [];
+private _magazinold = [];
+private _kosten = 0;
+
+//Datensatz aufschüsseln
+private _class = _Datensatz select 0;
+_airRaktenweapon = _Datensatz select 1;
+_airRaktenmagazin = _Datensatz select 2;
+_airGunweapon = _Datensatz select 3;
+_airGunmagazin = _Datensatz select 4;
+_vehWeapon = _Datensatz select 5;
+_vehMagazin = _Datensatz select 6;
+_pylon = _Datensatz select 7;
+_raketencontrol = _Datensatz select 8;
+_zusatz = _Datensatz select 9;
+_waffenkosten = _Datensatz select 10;
+
+//Fahrzeug erstellen
+private _heightOffset = 0.1;
+
+private _veh = createVehicle [_class, (position _box), [], 0, "NONE"];
```

Comment:
Ist an diesem Punkt schon sicher gestellt, dass die Position des Fahrzeugs frei von Hindernissen ist? Wenn ja kann man folgenden Trick verwenden:
In Arma ist es performanter (zumindest war das mal so), wenn man Fahrzeuge und Einheiten an Position 0,0,0 erstellt, da dort keine Abfragen angestellt werden, ob das erstellte Objekt mit etwas kollidieren würde.
Nachdem man es dort erstellt hat, kann man es einfach mit `setPos` an die gwünschte Stellöe beamen (und kann dabei auch direkt eine Höhe angeben).